Sources/SecureStore/KeyManagerService.swift

```diff
@@ -167,8 +180,9 @@ extension KeyManagerService {
         return encryptedString
     }
 
-    public func decryptDataWithPrivateKey(dataToDecrypt: String) throws -> String? {
-        let privateKeyRepresentation = try retrieveKeys().privateKey
+    public func decryptDataWithPrivateKey(dataToDecrypt: String,
```

Comment:
I'm not sure `public` is correct for this?

Reply:
Yep doesn't look like it, by that same analysis, does anything in the `KeyManagerService` need to be public apart from the `convenience init`..?

Reply:
I don't think so, no.
Does the initialiser even need to be public?